src/main/java/com/spotify/docker/client/messages/Info.java

```diff
@@ -223,20 +231,15 @@ public int hashCode() {
 
   @Override
   public String toString() {
-    return "Info{" +
-           "containers=" + containers +
-           ", images=" + images +
-           ", storageDriver='" + storageDriver + '\'' +
-           ", executionDriver='" + executionDriver + '\'' +
-           ", kernelVersion='" + kernelVersion + '\'' +
-           ", debug=" + debug +
-           ", fileDescriptors=" + fileDescriptors +
-           ", goroutines=" + goroutines +
-           ", eventsListener=" + eventsListener +
-           ", initPath='" + initPath + '\'' +
-           ", sockets=" + sockets +
-           ", memoryLimit=" + memoryLimit +
-           ", swapLimit=" + swapLimit +
-           '}';
+    return String.format("Info{ containers = %d, images = %d, storageDriver = %s, "
+                         + "driverStatus = %s, cpus = %d, memTotal = %d, name = %s, "
+                         + "executionDriver = %s, kernelVersion = %s, debug = %d, "
+                         + "fileDescriptors = %d, goroutines = %d, eventsListener = %d, "
+                         + "initPath = %s, initSha1 = %s, indexServerAddress = %s, "
+                         + "sockets = %s, memoryLimit = %b, swapLimit = %b",
+                         containers, images, storageDriver, driverStatus, cpus, memTotal, name,
+                         executionDriver, kernelVersion, debug, fileDescriptors, goroutines,
+                         eventsListener, initPath, initSha1, indexServerAddress,
+                         Joiner.on(", ").join(sockets), memoryLimit, swapLimit);
```

Comment:
Looks like I added it when originally implementing the `info` method. Maybe it existed in a previous version. I'll add a PR to just get rid of it.